simapp/simd/cmd/genaccounts.go

```diff
@@ -41,29 +40,33 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 		RunE: func(cmd *cobra.Command, args []string) error {
 			clientCtx := client.GetClientContextFromCmd(cmd)
 			depCdc := clientCtx.Codec
-			cdc := depCdc.(codec.Codec)
+			cdc := depCdc
```

Comment:
I think this is not needed at all actually. Just use `clientCtx.Codec` directly where needed